src/main/java/org/sonar/plugins/clojure/SonarProjectProperties.java

```diff
@@ -0,0 +1,52 @@
+package org.sonar.plugins.clojure;
```

Comment:
I think we don't need this file to have the properties you were looking for. Have a look at the `ClojureProperties.java`. That is one way of setting properties. You can find more information here: https://docs.sonarqube.org/display/DEV/API+Basics

Reply:
For some reason i cant get properties reading working that way. The properties doesnt even show in the web GUI (like instructions guide) where they can be admistrated in the global and project levels. 

I even tried register a property by using an annotation with no luck. 

Even your codes earlier file suffixes property (sonar.clojure.file.suffixes)  doesnt show in the web gui. You can check properties by logging as admin (admin/admin) and going to project -> administration -> general settings. There should be Clojure category with properties but for me it doesnt show up.

Perhaps @fsantiag you could try to get it working if there is some trivial problem which I cannot seem to be able to fix.

Disabling a sensor is a important feature because for example Eastwood  have some problems with macro heavy Clojure codebases.

Reply:
Interesting. Indeed the properties are not showing in the administrator view. I am not sure if we need to implement some custom class to display that. In any case, what you did is correct and the property `sonar.clojure.eastwood.disabled` is disabling eastwood successfully. I just ran locally and everything is fine. Great work :)
```
INFO: Sensor SonarClojure [clojure]
INFO: Eastwood plugin is disabled
```
We might need to have extra work to enable the configuration through the Web UI later. In any case, don't worry about that now.

Reply:
This seems to work only when using command line switch to the scanner -Dsonar.clojure.eastwood.disabled=true. Did you managed to get this working by setting this in sonar-project.properties file?

Reply:
Yes, I used the sonar-project.properties. Let try with a different project.

Reply:
Perhaps this is something which is different in my setup. Lets add command line alternative to the readme and resolve this part?

Reply:
Sounds good.